src/acceptance/app/dynamic_policy_test.go

```diff
@@ -377,3 +334,30 @@ var _ = Describe("AutoScaler dynamic policy", func() {
 		})
 	})
 })
+
+func concurrentHttpGet(count int, url string) {
+	client := &http.Client{
+		Timeout: 10 * time.Second,
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{
+				InsecureSkipVerify: true, //nolint:all
+			},
+		},
+	}
+
+	for i := 0; i < count; i++ {
+		go func() {
+			GinkgoWriter.Printf("[http get] [request] url: %s\n", url)
+
+			resp, err := client.Get(url) //nolint:all
+
+			if err != nil {
+				GinkgoWriter.Printf("[http get] [response] error: %s\n", err.Error())
+			}
```

Comment:
Maybe not needed in tests - but having had bad experiences:

```suggestion
			} else {
			  io.Copy(io.Discard, resp.Body)
			  defer resp.Body.Close()
			}

```

Reply:
Good catch! Thanks for pointing this out! 

Fixed with fbf0a26b38b805a7360119deb57bf18d67e5b299